extra/1-radio-stations.js

```diff
@@ -26,6 +33,19 @@
  */
 // `getStations` goes here
 
+function getStations(){
+  getAllFrequencies();
```

Comment:
this works because you declared `freqArray` outside of the `getAllFrequencies` function above - this is called a "side effect" and it generally considered a bad thing. Instead you should assign a variable within `getStations` to hold the value returned by `getAllFrequencies`
`
let freqArray = getAllFrequencies();`